recipes/el-secretario-mu4e

```diff
@@ -0,0 +1,4 @@
+(el-secretario-mu4e
+ :fetcher git
+ :url "https://git.sr.ht/~zetagon/el-secretario"
+ :files ("el-secretario-mu4e.el"))
```

Comment:
The byte-compile looks fixed now, but I'm realizing the checkdocs are not false positives (I should have checked more carefully):

`el-secretario-mu4e.el` with checkdoc 0.6.2:
```
el-secretario-mu4e.el:66: All variables and subroutines might as well have a documentation string
el-secretario-mu4e.el:90: All variables and subroutines might as well have a documentation string
el-secretario-mu4e.el:95: All variables and subroutines might as well have a documentation string
```